confluent_kafka/error.py

```diff
@@ -30,23 +29,25 @@ class ConsumeError(Exception):
     Args:
         error (KafkaError): The error that occurred.
```

Comment:
you should support exception chaining

Reply:
Don't think its supported until 3.0 

https://www.python.org/dev/peps/pep-3134/